### PR TITLE
Tree view

### PR DESCRIPTION
**Pull request by sumitarora**
should fix https://github.com/rangle/batarangle/issues/153

### CODE REVIEW

File: frontend/channel/backend-messaging-service.ts [hunk 1]
```diff
@@ -22,6 +22,9 @@ export class BackendMessagingService {
     });
 
     this.backgroundPageConnection.onMessage.addListener((message: any) => {
+      if (message.from && message.from === 'content-script') {
```
Comment (vanessayuenn) [inline frontend/channel/backend-messaging-service.ts]: Does this only occur on refresh?

Comment (sumitarora) [inline frontend/channel/backend-messaging-service.ts]: yes on reloading the app

Comment (vanessayuenn) [inline frontend/channel/backend-messaging-service.ts]: This actually happens every time any content loads, regardless of whether it's a refresh, or a new page load, or even if the application is one being inspected or not. Probably not ideal.

**Comment by vanessayuenn** [comment]
I would think there can be a better approach to message passing on reload, but we can tackle that later down the road. :shipit:

**Pull request merged**